getting_started/manage_your_plan/fuse_online.adoc

```diff
@@ -0,0 +1,22 @@
+[[getting-started-manage-your-fuse-plan]]
+= Manage Your Fuse Online Plan
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+[[customize-your-fuse-plan-with-add-ons]]
+== Customize Your Plan
+
+You can customize your plan with add-ons.
```

Comment:
As I understand the fuse online requirements, this is actually not the case. The user can choose among preconfigured plans but cannot customize them.